packages/web/src/pages/artist-dashboard-page/ArtistDashboardPage.tsx

```diff
@@ -82,8 +85,12 @@ export const ArtistDashboardPage = () => {
     makeGetDashboard()
   )
   const listenData = useSelector(getDashboardListenData)
-  const status = useSelector(getDashboardStatus)
+  const dashboardStatus = useSelector(getDashboardStatus)
   const isMatrix = useSelector(getTheme) === Theme.MATRIX
+  const { data, status: balanceStatus } = useUSDCBalance()
+  const balance =
+    (data?.toNumber() ?? 0) / 10 ** TOKEN_LISTING_MAP.USDC.decimals
```

Comment:
Is this how we do it for audio? Not that I expect people to have huge amounts of USDC. But I think it's probably best to do the decimal math while still in BN space, then convert to number.